samples/cpp/benchmark_app/utils.cpp

```diff
@@ -178,8 +176,8 @@ std::map<std::string, std::string> parse_value_per_device(const std::vector<std:
             if (it != devices.end()) {
                 result[device_name] = nstreams;
             } else {
-                throw std::logic_error("Can't set nstreams value " + std::string(nstreams) + " for device '" +
-                                       device_name + "'! Incorrect device name!");
+                throw std::logic_error("Can't set nstreams/infer_precision value " + std::string(nstreams) +
```

Comment:
It is not clear from this message what is wrong actually. It is better avoid printing parameter names `nstreams/infer_precision`  in common function